custom_components/spotcast/__init__.py

```diff
@@ -253,7 +253,17 @@ def play(client, spotify_device_id, uri, random_song, repeat, shuffle, position)
             uri,
             spotify_device_id,
         )
-        if uri.find("track") > 0:
+        if uri.find("show") > 0:
+            show_episodes_info = client.show_episodes(uri)
+            if show_episodes_info and len(show_episodes_info["items"]) > 0:
```

Comment:
```suggestion
            if show_episodes_info and show_episodes_info["items"]:
```

Reply:
Saw your review now! Thanks, I'm nackered now after a whole day fixing. Can you open a new PR?